### PR TITLE
[DDW-1076] Wallet name update feature issues

### PR DESCRIPTION
**Pull request by DeeJayElly**
This PR fixes wallet name length when the name of the wallet is too big. Instead of presenting everywhere on smaller screen resolutions full length of the name, we are shrinking it so it can fit the screen and the container.

## Screenshots

- Screenshot

Screenshot 2019-11-15 15 54 34
Screenshot 2019-11-15 15 30 50
Screenshot 2019-11-15 15 30 25

---

## Testing Checklist

- [Slack QA thread](https://input-output-rnd.slack.com/archives/GGKFXSKC6/p1573830301048300)
- [ ] When the wallet name is larger, test if its presented well on all screens sizes.

---

### Basics

- [ ] PR has been assigned and has appropriate labels (`feature`/`bug`/`chore`, `release-x.x.x`)
- [ ] PR is updated to the most recent version of the target branch (and there are no conflicts)
- [ ] PR has a good description that summarizes all changes and shows some screenshots or animated GIFs of important UI changes
- [ ] CHANGELOG entry has been added to the top of the appropriate section (*Features*, *Fixes*, *Chores*) and is linked to the correct PR on GitHub
- [ ] Automated tests: All acceptance and unit tests are passing (`yarn test`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *development* build (`yarn dev`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *production* build (`yarn package` / CI builds)
- [ ] There are no *flow* errors or warnings (`yarn flow:test`)
- [ ] There are no *lint* errors or warnings (`yarn lint`)
- [ ] There are no *prettier* errors or warnings (`yarn prettier:check`)
- [ ] There are no missing translations (running `yarn manage:translations` produces no changes)
- [ ] Text changes are proofread and approved (Jane Wild / Amy Reeve)
- [ ] Japanese text changes are proofread and approved (Junko Oda)
- [ ] UI changes look good in all themes (Alexander Rukin)
- [ ] Storybook works and no stories are broken (`yarn storybook`)
- [ ] In case of dependency changes `yarn.lock` file is updated

### Code Quality
- [ ] Important parts of the code are properly commented and documented
- [ ] Code is properly typed with flow
- [ ] React components are split-up enough to avoid unnecessary re-renderings
- [ ] Any code that only works in main process is neatly separated from components

### Testing
- [ ] New feature/change is covered by acceptance tests
- [ ] New feature/change is manually tested and approved by QA team
- [ ] All existing acceptance tests are still up-to-date
- [ ] New feature/change is covered by Daedalus Testing scenario
- [ ] All existing Daedalus Testing scenarios are still up-to-date

### After Review
- [ ] Merge the PR
- [ ] Delete the source branch
- [ ] Move the ticket to `done` column on the YouTrack board
- [ ] Update Slack QA thread by marking it with a green checkmark

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -48,6 +48,7 @@ Changelog
 
 ### Fixes
 
+- Fixed Wallet name length issues on smaller screen sizes ([PR 1689](https://github.com/input-output-hk/daedalus/pull/1689))
```
Comment (nikolaglumac) [inline CHANGELOG.md]: @DeeJayElly this entry should go under new section "vNext" which you should add on top of the file like I did in my PR: https://github.com/input-output-hk/daedalus/pull/1690/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR4-R9

**Comment by IuliaDolishniak** [comment]
@DeeJayElly @nikolaglumac may be we should also not allow here more than 40 symbols?
screen

**Comment by nikolaglumac** [comment]
No @IuliaDolishniak - this part is fine.

**Comment by a-rukin** [comment]
image
uh... found this bug -_-

File: CHANGELOG.md (hunk 1 above)
Comment (DeeJayElly) [reply CHANGELOG.md]: Fixed

**Comment by DeeJayElly** [comment]
> image
> 
> uh... found this bug -_-

Fixed

**Comment by IuliaDolishniak** [comment]
this isssue still not fixed
screen

**Comment by IuliaDolishniak** [comment]
elipsis are not shown 
screen

**Comment by IuliaDolishniak** [comment]
Part of the symbol is truncated
screen

**Comment by IuliaDolishniak** [comment]
screen

File: source/renderer/app/components/wallet/settings/WalletSettings.js [hunk 2]
```diff
@@ -82,6 +82,7 @@ type Props = {
 
 type State = {
   isFormBlocked: boolean,
+  walletNameFieldMaxlength: string,
```
Comment (tomothespian) [inline source/renderer/app/components/wallet/settings/WalletSettings.js]: @DeeJayElly  Please include length as a number!
ReactPolymorph (Input): https://github.com/input-output-hk/react-polymorph/blob/develop/source/components/Input.js#L25

**Comment by tomothespian** [comment]
@IuliaDolishniak everything you reported is fixed. Can you please review again?

**Comment by IuliaDolishniak** [comment]
@tomothespian waiting for the build

**Pull request merged**